### PR TITLE
Paypal Checkout Demo

### PR DESCRIPTION
**Pull request by jcnoriega**
### Summary of changes

 - Added demo App for PayPalIntegration. Once #22 is accepted, this PR will be merged to it and then all into the main branch

 ### Checklist
ezgif com-gif-maker

- @jcnoriega

### CODE REVIEW

File: Demo/build.gradle [hunk 1]
```diff
@@ -70,6 +70,7 @@ dependencies {
     implementation deps.composeRuntimeLivedata
     implementation deps.composeFoundation
     implementation deps.composeMaterial
+    implementation "androidx.constraintlayout:constraintlayout-compose:1.0.0-alpha07"
```
Comment (andrespelaezp) [inline Demo/build.gradle]: can you match this line with the dependencies style?

File: Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt [hunk 2]
```diff
@@ -55,15 +68,134 @@ class PayPalFragment : Fragment() {
...
+        }
+
+    }
+
+    @Composable
+    private fun CheckoutSuccess(result: PayPalCheckoutResult.Success, modifier: Modifier) {
+        Column(modifier = modifier.padding(horizontal = 16.dp)) {
+            Text(
+                text = "Order Approved",
+                fontSize = 24.sp,
```
Comment (andrespelaezp) [inline Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt]: qq: Can Compose get these strings values out of the _strings.xml_? and is it worth doing it for the Demo app?

Comment (jcnoriega) [reply Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt]: yes it can. I'll add them

File: Demo/build.gradle (hunk 1 above)
Comment (jcnoriega) [reply Demo/build.gradle]: yep, i forgot hehe

File: Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt (hunk 2 above)
Comment (jcnoriega) [reply Demo/src/main/java/com/paypal/android/ui/paypal/PayPalFragment.kt]: @andrespelaezp fixed

**Review by andrespelaezp — APPROVED** [review]
lgtm!

**Pull request merged**